test/core/backend/BackendController.test.js

```diff
@@ -72,29 +72,6 @@ describe('Backend', () => {
         .be.eql(definition.actions.sayHello);
     });
 
-    it('should rejects if the controller definition is invalid', () => {
```

Comment:
Shouldn't this test be updated rather than deleted?

Reply:
We are not checking the definition validity here anymore because the check require a promise and we will loose a lot of DX if developers have to declare controllers with `await app.controller.use`